src/traces/image/attributes.js

```diff
@@ -51,6 +51,16 @@ module.exports = extendFlat({
             'otherwise it defaults to `rgb`.'
         ].join(' ')
     },
+    interpolate: {
+        valType: 'boolean',
```

Comment:
L

Reply:
... as one may add other types of interpolation later.

Reply:
With values `[false, true]`? Or must values be str?

Reply:
It should be strings e.g. 'linear' and 'nearest'. You could also use space in each key if it has multiple words.

Reply:
done